include/openspace/scene/profile.h

```diff
@@ -164,20 +163,19 @@ class Profile {
      * \return The Lua library that contains all Lua functions available for profiles
      */
     static scripting::LuaLibrary luaLibrary();
+
+    std::string file_subName_assets = "";
+    std::string assetFileExtension = ".asset";
 };
 
 /**
- * This function takes a profile and returns its asset-ifyied version as a string. This
- * is the format that is saved as a scene file that, in turn, is provided to OpenSpace as
- * the root asset to load. This function is a key step to be able to load a Profile in
- * OpenSpace (at the moment).
- * 
- * \param profile The profile that should be converted to the asset-file format
- * 
- * \return The string representation of the provided profile, ready to be loaded as an
- *         asset
+ * Function to retrieve the individual additional script lines included in the profile
+ *
+ * \param p The profile that should be processed
+ *
+ * \return vector of strings containing the individual script lines
  */
-std::string convertToScene(const Profile& profile);
+std::vector<std::string>& additionalScripts(Profile& p);
```

Comment:
This function is (correctly) not called anywhere and can be removed entirely

Reply:
Gone